x-pack/plugins/enterprise_search/public/applications/app_search/app_logic.ts

```diff
@@ -41,6 +43,10 @@ export const AppLogic = kea<MakeLogicType<AppValues, AppActions, Required<Initia
       },
     ],
     configuredLimits: [props.configuredLimits.appSearch, {}],
+    showGateForm: [
+      props.appSearch.kibanaIsEnabled === false && props.appSearch.role.roleType === 'owner',
+      false,
+    ],
```

Comment:
`showGateForm` is a boolean but is used as an array here. is it intentional?

Reply:
This is me, not knowing Kea very well. :) Thanks for pointing that out.